api.go

```diff
@@ -469,9 +469,8 @@ func NewAPI(config Config, a Adapter) API {
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
     <title>` + title + `</title>
     <!-- Embed elements Elements via Web Component -->
-    <link href="https://unpkg.com/@stoplight/elements@8.1.0/styles.min.css" rel="stylesheet" />
-    <script src="https://unpkg.com/@stoplight/elements@8.1.0/web-components.min.js"
-            integrity="sha256-985sDMZYbGa0LDS8jYmC4VbkVlh7DZ0TWejFv+raZII="
+    <link href="https://unpkg.com/@stoplight/elements/styles.min.css" rel="stylesheet" />
+    <script src="https://unpkg.com/@stoplight/elements/web-components.min.js"
```

Comment:
Thanks, but we will still want to lock to a specific version. We've had releases break things before and this is just safer for various reasons!

```suggestion
    <script src="https://unpkg.com/@stoplight/elements@9.0.0/web-components.min.js" integrity="sha256-p7xyydU1NAgHE+zXB+R/zjfj6HEczZCjqipItbyIp6I="
```